### PR TITLE
Project 1 [To Do List App]

### PR DESCRIPTION
**Pull request by siddhant2002**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added issue number to the PR.
- [x] I have added screen recording of the working app.

https://github.com/anishaswain/Simple-JS-Projects/assets/78552027/c85a4664-444b-44fc-8b7e-bbb435ce0df8

### CODE REVIEW

**Comment by siddhant2002** [comment]
Issue number #1

File: project1[To_Do_List]/alpha.js
Comment (anishaswain) [inline project1[To_Do_List]/alpha.js:1]: Please add another folder with you github username inside project1 folder and move your js/html/css files inside it (Please go through the redme file carefully)
- squash the commits into one commit and add meaningful commit messge

**Review by anishaswain — CHANGES_REQUESTED** [review]

**Comment by siddhant2002** [comment]
Please check the pr

**Review by anishaswain — CHANGES_REQUESTED** [review]
@siddhant2002 nice work. Please squash the commits into one commit and add meaningful commit message ([How to squash multiple commit messages](https://www.freecodecamp.org/news/git-squash-commits/))

**Comment by siddhant2002** [comment]
@anishaswain  Please check the pr

**Comment by anishaswain** [comment]
> Require a

@siddhant2002 there are 17 commits on your PR now. After squash there should only be 1 commit

**Comment by siddhant2002** [comment]
@anishaswain Can I raise another pr with 1 commit. It has taken all pull requests as commits.

**Comment by anishaswain** [comment]
> @anishaswain Can I raise another pr with 1 commit. It has taken all pull requests as commits.

Sure

**Pull request closed without merging**